webapp/tests/Unit/Controller/Team/ProblemControllerTest.php

```diff
@@ -183,4 +184,40 @@ public function testInteractiveSamples(): void
         $response = $this->client->getResponse();
         self::assertEquals(404, $response->getStatusCode());
     }
+
+    /**
+     * Test that the problems page does not show page, statement, sample data before contest start.
+     */
+    public function testAccessProblemBeforeContestStarts(): void
+    {
+        /** @var EntityManagerInterface $em */
+        $em = self::getContainer()->get(EntityManagerInterface::class);
+
+        $contest = $em->getRepository(Contest::class)->findOneBy(['shortname' => 'demo']);
+        $originalStartTime = $contest->getStarttimeString();
+
+        $lastSpacePosition = strrpos($originalStartTime, ' ');
+        $datetime = substr($originalStartTime, 0, $lastSpacePosition);
+        $timezone = substr($originalStartTime, $lastSpacePosition + 1);
+
+        $date = new \DateTime($datetime, new \DateTimeZone($timezone));
+        $date->modify('+1 hour');
```

Comment:
Can you do this with a `Fixture` like https://github.com/DOMjudge/domjudge/blob/main/webapp/src/DataFixtures/Test/DemoAboutToStartContestFixture.php? That makes it much easier to see what kind of state this test runs from.

Reply:
ok done

Reply:
I don't see the Fixture yet. Did you push the change?